apps/src/templates/teacherNavigation/teacher-navigation.module.scss

```diff
@@ -9,13 +9,13 @@
 
 .pageWithHeader {
   flex-grow: 1;
-  padding: 20px;
+  padding: 20px 48px 20px 38px;
```

Comment:
I'm tempted to add more padding at the bottom here. Most pages already have some, but for others only 20px might make it cramped. What do you think?

Reply:
I added a little, which I think will help, but I am also trusting that to work itself out during a future bug bash.